community-operators/rook/rook.v0.1.0.csv.k8s.yaml

```diff
@@ -0,0 +1,306 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  creationTimestamp: null
+  name: rook.v0.1.0
+  namespace: placeholder
+  annotations:
+    tectonic-visibility: ocs
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "ceph.rook.io/v1",
+          "kind": "CephCluster",
+          "metadata": {
+            "name": "my-rook-ceph"
+            "namespace": "rook-ceph"
+          },
+          "spec": {
+            "cephVersion": {
+              "image": "ceph/daemon-base:latest-master"
+            },
+            "dataDirHostPath": "/var/lib/rook",
+            "mon": {
+              "count": "3"
+            },
+            "dashboard": {
+              "enabled": true
+            },
+            "network": {
+              "hostNetwork": false
+            },
+            "rbdMirroring": {
+              "workers": "0"
+            },
+            "storage": {
+              "useAllNodes": true
+              "useAllDevices": true
+            },
+          }
+        },
+      ]
+spec:
+  displayName: rook
+  description: |
+      Rook: Storage Orchestration for Kubernetes
+
+      Rook runs as a cloud-native service for optimal integration with applications in need of storage, and handles the heavy-lifting behind the scenes such as provisioning and management.      ## Before Your Start
+      Rook orchestrates battle-tested open-source storage technologies including Ceph, which has years of production deployments and runs some of the worlds largest clusters.
+      Rook is open source software released under the Apache 2.0 license. Rook has an active developer and user community.
+
+  keywords: ['rook', 'ceph', 'storage', 'object storage', 'open source']
+  version: 0.1.0
+  minKubeVersion: 1.11.0
+  maturity: alpha
+  maintainers:
+  - name: Red Hat, Inc.
+    email: customerservice@redhat.com
+  provider:
+    name: Red Hat, Inc.
+  labels:
+    alm-owner-etcd: etcdoperator
+    operated-by: rookoperator
+  selector:
+    matchLabels:
+      alm-owner-etcd: etcdoperator
+      operated-by: rookoperator
+  links:
+  - name: Blog
+    url: https://blog.rook.io
+  - name: Documentation
+    url: https://rook.github.io/docs/rook/master/
+  - name: rook Operator Source Code
+    url: https://github.com/rook/rook/tree/master/pkg/operator/ceph
+
+  icon:
+  - base64data: #TODO
```

Comment:
If the icon is pending, could this PR have a hold? This can also be added in a future PR, but the `icon` field should be removed until then.